plugins/event_notification/lib/model/BatchEventNotificationTemplate.php

```diff
@@ -116,8 +116,39 @@ protected function addEventNotificationDispatchJob($eventNotificationType, kEven
 		
 		$batchJob = kJobsManager::addJob($batchJob, $jobData, $jobType, $eventNotificationType);
 		$jobData->setJobId($batchJob->getId());
+
+		$batchJobStatus = BatchJob::BATCHJOB_STATUS_PENDING;
+		if ($this->areDelayedEventConditionsMet($jobData, $entryId))
+		{
+			$batchJobStatus = BatchJob::BATCHJOB_STATUS_DELAYED;
+		}
+
 		$batchJob->setData($jobData);
-		
-		return kJobsManager::updateBatchJob($batchJob, BatchJob::BATCHJOB_STATUS_PENDING);
+		return kJobsManager::updateBatchJob($batchJob, $batchJobStatus);
+	}
+
+	public function areDelayedEventConditionsMet(&$jobData, $objectId = null)
+	{
+		$delayedEventConditions = $this->getEventDelayedConditions();
```

Comment:
is delayed conditions (getEventDelayedConditions) meant to be an array of condition types / enum ? 
if it's meant to be just one condition then better give it a singular name, otherwise it's confusing